src/public/app/widgets/note_tree.js

```diff
@@ -279,6 +279,7 @@ export default class NoteTreeWidget extends TabAwareWidget {
             source: treeData,
             scrollParent: this.$tree,
             minExpandLevel: 2, // root can't be collapsed
+            tooltip: (options.get('treeNoteTitleTooltipEnabled') === 'true'),
```

Comment:
you can just use `options.is('treeNoteTitleTooltipEnabled')`

Reply:
Sure, will do 👍